docs/app-debug/mpir-tools.rst

```diff
@@ -0,0 +1,100 @@
+Using MPIR-based tools with Open MPI
```

Comment:
This section appears to be all-new.  I have a few top-level thoughts:

1. We should probably explain what MPIR is, and that it has been replaced by the PMIx tool attach thingy blah blah blah (I assume that this is the kind of info you are waiting for from Ralph).  But since so many legacy tools still use MPIR, there's a way to use these tools: the MPIR shim.
2. Have a subsection on what the MPIR shim is (no more than 1-3 sentences, I would think -- let the MPIR shim web site do the bulk of the explaining), and a link out to the MPIR shim web site.
3. Have a subsection on how to build/install Open MPI with the MPIR shim.  
    * In the "validation" section, you have a bullet list of installation points.  It might be ok to replace most of that with a code block with the individual commands in a code block, somewhat like an annotated shell script (vs. a bullet for each command and explanation of that command).
4. Have a subsection about how to validate that the MPIR shim is working (e.g., export the shell variables, compile the test MPI program, run it, attach to it, ... etc.).